spec/requests/entrypoint_spec.rb

```diff
@@ -74,6 +74,25 @@
     )
   end
 
+  context 'UI is available' do
+    it 'product_info contains branding_info' do
+      api_basic_authorize
+
+      expect(ActionController::Base.helpers).to receive(:image_path).at_least(2).times.and_return("foo")
```

Comment:
Should the test above be modified or a new test added where this is not stubbed?

Reply:
I'll update the test above!